src/main/java/net/library/model/dto/response/HttpCidSuccessResponse.java

```diff
@@ -1,4 +1,4 @@
-package net.library.exception;
+package net.library.model.dto.response;
```

Comment:
It is incorrect structure/architecture of your code. You should return the basic type of controller. For example for BookController you should return book. But Book dto may extend Success response for example. But make a better name that Success response